risc0/zkp/src/prove/accum.rs

```diff
@@ -101,15 +101,15 @@ impl<'a, F: Field> CircuitStepHandler<F::Elem> for Handler<'a, F> {
     ) -> Result<()> {
         assert!(cycle < self.cycles);
         match name {
-            "plonkWriteAccum" => {
+            "WriteAccumulator" => {
```

Comment:
Unfortunately we can't change these until we make a corresponding change in the circuit.

Reply:
Marking this as resolved in order to focus attention on the question of whether the naming suggestion improves clarity; will worry about breaking changes later